dotcom-rendering/src/lib/assets.ts

```diff
@@ -15,16 +15,24 @@ try {
 	// do nothing
 }
 
-// TODO: this should be removed in favour of `frontendAssetsFullURL` defined in CAPI
```

Comment:
I'm guessing this this TODO no longer relevant?

Reply:
We could use the CAPI value but we still need to do something different when running locally so in a way it's easier to keep all the logic here. It also means we don't need to pass CAPI data into this code, keeping things simple.